<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tool_add</class>
 <widget class="QWidget" name="tool_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>160</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="Tool_Name">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tool Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="Tool_Name_val"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="X_Off">
        <property name="text">
         <string>X-Offset</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="X_Off_val"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Y_Off">
        <property name="text">
         <string>Y-Offset</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="Y_Off_val"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Z_Off">
        <property name="text">
         <string>Z-Offset</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="Z_Off_val"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="Add_Tool">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add Tool</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Modify Tool</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Remove_Tool">
      <property name="text">
       <string>Remove Tool</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="Tool_List"/>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Exit">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Tool_Picture">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>411</width>
     <height>471</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
